arquillian-recorder-reporter/arquillian-recorder-reporter-api/src/main/java/org/arquillian/recorder/reporter/ReporterConfiguration.java

```diff
@@ -213,7 +213,7 @@ public void validate() throws ReporterConfigurationException {
         } catch (IllegalArgumentException ex) {
             throw new ReporterConfigurationException(
                 "Report frequency you specified in arquillian.xml is not valid. "
-                    + "Supported frequencies are: " + ReportFrequency.getAll());
```

Comment:
add here report frequency configured in arq.xml so user knows what he configured wrong